src/imp/libc/c.rs

```diff
@@ -58,3 +58,71 @@ const SD_BOTH: c_int = 2;
 pub(crate) const SHUT_WR: c_int = SD_SEND;
 pub(crate) const SHUT_RD: c_int = SD_RECEIVE;
 pub(crate) const SHUT_RDWR: c_int = SD_BOTH;
+
+pub(crate) use winapi::shared::ws2def::LPWSABUF;
+
+pub(crate) unsafe fn sendmsg(
+    s: SOCKET,
+    lpBuffers: LPWSABUF,
+    dwBufferCount: c_ulong,
+    lpTo: *mut sockaddr,
+    iToLen: c_int,
+    dwFlags: c_ulong,
+) -> c_int {
+    let mut lpNumberOfBytesSent: c_ulong = 0;
+
+    // No overlapping IO support
+    let lpOverlapped = core::ptr::null_mut();
+    let lpCompletionRoutine = None;
+
+    let res = WSASendTo(
```

Comment:
I haven't found much information on best practices, on which one to use over the other. Will do some more research before setteling on one.

Reply:
In case it helps, another option would be to omit `sendmsg` and `recvmsg` on Windows entirely here.

In places where Windows has significantly different APIs from POSIX/etc., rustix tends to just omit the Windows APIs, rather than trying to coerce them into POSIX-like abstractions. In such cases, Windows/POSIX portability is often better handled at higher levels of abstraction. I don't know if that's the case here, but I wanted to mention this option in case it is.

Reply:
I do like having support, so trying to make sure it works in all cases. Added some docs now as to why I am using the calls that I am using